NEXT_CHANGELOG.md

```diff
@@ -24,328 +24,3 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 -->
 
 # [x.x.x] (unreleased) - 2022-mm-dd
-## ❗ BREAKING ❗
```

Comment:
We could now add an experimental section in the example snippet above

Reply:
Fixed